Xamarin.Forms.Platform.UAP/FormsTextBoxStyle.xaml

```diff
@@ -211,17 +211,17 @@
 						                  ContentTemplate="{TemplateBinding HeaderTemplate}" Content="{TemplateBinding Header}"
 						                  Foreground="{ThemeResource SystemControlForegroundBaseHighBrush}" FontWeight="Normal"
 						                  Margin="0,0,0,8" Grid.Row="0" Visibility="Collapsed" x:DeferLoadStrategy="Lazy" />
-						<ScrollViewer x:Name="ContentElement" AutomationProperties.AccessibilityView="Raw"
-                                     
+                        <ScrollViewer AutomationProperties.AccessibilityView="Raw"
 						              HorizontalScrollMode="{TemplateBinding ScrollViewer.HorizontalScrollMode}"
 						              HorizontalScrollBarVisibility="{TemplateBinding ScrollViewer.HorizontalScrollBarVisibility}"
 						              IsTabStop="False" IsHorizontalRailEnabled="{TemplateBinding ScrollViewer.IsHorizontalRailEnabled}"
 						              IsVerticalRailEnabled="{TemplateBinding ScrollViewer.IsVerticalRailEnabled}"
 						              IsDeferredScrollingEnabled="{TemplateBinding ScrollViewer.IsDeferredScrollingEnabled}"
 						              Margin="{TemplateBinding BorderThickness}" Padding="{TemplateBinding Padding}" Grid.Row="1"
 						              VerticalScrollBarVisibility="{TemplateBinding ScrollViewer.VerticalScrollBarVisibility}"
-						              VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}" ZoomMode="Disabled"
-                                      VerticalAlignment="{TemplateBinding VerticalContentAlignment}" />
+						              VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}" ZoomMode="Disabled">
+                            <ContentPresenter x:Name="ContentElement" VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}" /> 
```

Comment:
@hartez This content presenter is the only difference between the two.  That is what was added to fix the text not showing right away but caused the height measure to then work incorrectly.

Reply:
that line and which element is labeled "ContentElement" is the only difference.

I was hoping to be able to pull out the ContentPresentor in OnTemplateApply in the FormsTextBox to avoid the extra template, but I don't think the template is created by the time it's needed to measure.

Also a note on the ScrollViewer as the Content Element, I made a blank UWP project and pulled up the control template, the ScrollViewer is in there as the Content Element in a stock template.

Reply:
Okay, I see it. I'll try to test and review tomorrow. 

So with this change, does anything still use the other control template (FormsTextBoxStyle)?

Reply:
yes, that is used by all the displayed text boxes...the "Measure" one is only used by the off visual tree textbox used for measuring